### DISCUSSION TITLE
Dutch translations of Seaside

### DISCUSSION THREAD

**Discussion by haiwatta** [sumpfork/dominiontabs#462] [General]
Hi,

I noticed some cards of Dominion: seaside are not translated in Dutch.

I have no clue on how to change the translations, so I though to write a message.
These are the cards not translated. I will mention the English name, then the dutch name and the dutch text on the card.
I hope it makes sense. I used the text including including next-lines as on the original card. With +x for money I mentioned that in text, so that have to be replaced by the money-logo.
When in doubt, also check the photo's.

Astrolab
Sextant

Nu en  aan het begin van je 
volgende beurt 
+1 (money) 
+1 Aanschaf

Blockade
Blokkade

Pak been kaart die ten hoogste 
4 kost en leg die opzij.
Aan het begin van  je volgende 
beurt: doe de kaart in je hand.
Terwijl de kaart opzijgelegd is:
Pakt een andere speler er in zijn
beurt een exemplaar van, dan pakt
hij een vloek.

Corsair
Zeerover

+2 (money)

Aan het begin van je 
volgende beurt: +1 kaart.
Toto dan vernietigt iedere andere
speler het eerste Zilver of Goud 
dat hij in zijn beurt speelt.

Monkey
Aap

Tot je volgende beurt:
Als de speler rechts van
je een kaart pakt: 
+1 kaart

Aan het begin van je
volgende beurt:
+1 kaart

Pirate
Piraat

Aan het begin van je volgende
beurt: pak een geldkaart die ten
hoogste 6 (money) kost en neem die in
je hand
______________________________
Als een speler een geldkaart 
pakt, mag je deze kaart uit je
hand spelen

Sailor
Matroos

+1 actie
Eenmaal in deze beurt: als je
een duurzame kaart pakt, mag je
die spelen.

Aan het begin van je volgende
beurt: +2 (money) en je mag een kaart uit je hand vernietigen

Sea chart
Zeekaart

+1 kaart
+1 actie

Toon de bovenste kaart van je
trekstapel. Heb je er een exemplaar
van in het spel, dan neem je de
getoonde kaart in je hand

Tide pools
Getijdenpoelen

+3 kaarten
+1 actie

Aan het begin van je volgende
beurt: leg 2 kaarten af.

![20230406_131523](https://user-images.githubusercontent.com/4047378/23036
![20230406_131556](https://user-images.githubusercontent.com/4047378/230368842-0fcda773-7ed5-4a2e-aaba-1e0a34e39e4f.jpg)
8541-cbd2a36a-a2b0-42e8-8b26-d62605f0cca2.jpg)
![20230406_131647](https://user-images.githubusercontent.com/4047378/230369110-28c702c8-666b-484c-a709-45a4a9fba7aa.jpg)
![20230406_131809](https://user-images.githubusercontent.com/4047378/230369271-7071dccb-40df-4765-9
![20230406_131838](https://user-images.githubusercontent.com/4047378/230369341-3228c337-6286-4182-ae46-c929259d2df1.jpg)
ae0-70ae7c5cade6.jpg)
![20230406_131911](https://user-images.githubusercontent.com/4047378/230369452-36d772dc-9282-4ab0-afbc-ebf491c0d2fd.jpg)

![20230406_132006](https://user-images.githubusercontent.com/4047378/230369514-bcafa553-13e7-419b-87dd-e2089e3624c3.jpg)
![20230406_132037](https://user-images.githubusercontent.com/4047378/230369560-62d284ea-fe58-4033-99a9-dea871a76c07.jpg)
![20230406_131523](https://user-images.githubusercontent.com/4047378/230370399-d5bf4af5-53be-4df6-a93e-369fe83f9160.jpg)
![20230406_131809](https://user-images.githubusercontent.com/4047378/230370445-82e130bf-5e76-462f-863c-773d28c4d802.jpg)

**Comment by sumpfork** [comment]
If someone wants to format this as required and submit a PR I'm happy to accept it, but I likely won't have time anytime soon.
